### PR TITLE
tests: add jenkins build status

### PR DESCRIPTION
**Pull request by erikkn**
Get the build job status and add it to the document file name.The value is used by Logstash to add this as as an attribute to all the documents.

### CODE REVIEW

**Comment by erikkn** [comment]
ok to test

**Comment by erikkn** [comment]
@mxinden

File: tests/jenkins-jobs/scripts/log-analyzer-copy.sh [hunk 1]
```diff
@@ -27,6 +27,11 @@ fi
 SOURCE_BRANCH=${BRANCH_NAME//[=,]/_}
 TARGET_BRANCH=${CHANGE_TARGET//[=,]/_}
 
+#Check whether the BUILD_STATUS is not empty, if add it to the additional fields
```
Comment (mxinden) [inline tests/jenkins-jobs/scripts/log-analyzer-copy.sh]: Please update the comment

File: tests/jenkins-jobs/scripts/log-analyzer-copy.sh [hunk 2]
```diff
@@ -27,6 +27,11 @@ fi
...
+if [ -n "${BUILD_RESULT}" ]; then
+   additionalFields="${additionalFields}build_status=${BUILD_RESULT},"
```
Comment (mxinden) [inline tests/jenkins-jobs/scripts/log-analyzer-copy.sh]: I would like this to be consistent. Either use `result` or `status` everywhere. As Jenkins chooses the keyword `result`, maybe stick with that.

**Comment by mxinden** [comment]
ok to test

**Comment by erikkn** [comment]
ok to test

**Comment by erikkn** [comment]
AWS failed, this is not related and all the other succeeded. Approved by Max.

**Pull request merged**